### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by carlos-ssh**
### We created the main instructions for the players including the logical of the game:
- Added steps needed to play the game.
- What the players need to do.
- How the game should display for players.

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑 

## What a remarkable project you have built 💯! This is impressive 🤩👌🧨!

### But in order to get it approved, we need to work on some changes. Let's see...

- [ ] `main.rb` uses `puts/print` and `get.chomp` commands. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/main_rb.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

You are allowed to use inline comments but your game needs to run too. What you can do is convert the non-logic part to actual code - taking player names, displaying board, informing players if they win/lose/draw. You can use inline comments or simply `puts` to mock the logic. Use any loop to let each player take turns.

Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

*Need help? Shoot me a message on Slack - [@Ghosh](https://app.slack.com/client/T47CT8XPG/DRDM6LAQN/user_profile/URB1MHXK6)*
<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Comment by carlos-ssh** [comment]
-Add in main.rb uses puts/print and get.chomp commands.

File: bin/main.rb [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+
+puts "\n  1 | 2 | 3\n  --|---|--\n  4 | 5 | 6\n  --|---|--\n  7 | 8 | 9"
+
+puts "\nNow your move is displayed!"
+
+puts "\nAnd it's next player move:"
+
+gets
+
+puts "\nThe player that gets 3 symbols in a row wins"
```
Comment (expjazz) [inline bin/main.rb:71]: - [optional ] All of this should be inside the loop

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

Hello Team :+1: 

You did a really nice job here. The way you outlined makes clear that you know what you need to do. And now that you have this plan, the next milestone will be really easier for you :100: 

However, there is one requirement that should have been appointed before and it was not. Since this is the case, I will mark as optional:

- [ optional ] The turns should be wrapped in a loop.

image

#### You can move on to the next milestones.

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**